{
    "type": "bundle",
    "id": "bundle--fbbf66ca-9a05-43dd-b941-d319510cbc8c",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--c3ad0b48-37e5-4675-93eb-87daf9a20c3c",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-02-29T00:27:08.540035Z",
            "modified": "2024-02-29T00:27:08.540035Z",
            "name": "CVE-2021-46995",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\ncan: mcp251xfd: mcp251xfd_probe(): fix an error pointer dereference in probe\n\nWhen we converted this code to use dev_err_probe() we accidentally\nremoved a return. It means that if devm_clk_get() it will lead to an\nOops when we call clk_get_rate() on the next line.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2021-46995"
                }
            ]
        }
    ]
}